client/src/components/editor/EditorHome.js

```diff
@@ -1,17 +1,24 @@
 import React, {useState} from "react"
-import {Container,Accordion, Card, InputGroup, Form, Button, Modal} from "react-bootstrap"
+import {Container,Accordion, Card, InputGroup, Form, Button, Modal, Navbar, Nav, Carousel} from "react-bootstrap"
 import {BsFillPlusCircleFill} from "react-icons/bs";
 import toast, { Toaster } from 'react-hot-toast';
 
 import "./editorHome.css"
+import HomeSectionCard from  "./HomeSectionCard.js"
 
 const EditorHome = () => {
 
     const [show, setShow] = useState(false);
-
     const handleClose = () => setShow(false);
     const handleShow = () => setShow(true);
-    
+    const [header, setHeader] = useState(null);
+    const [sHeader, setSHeader] = useState(null);
```

Comment:
In the static analyzer sHeader is not used. You can delete this part if it's not used.

Reply:
The sHeader, which is the subheader is currently not a section used in the editor. However, when we later decide upon its use on the client home page, it will be used.

Reply:
Because unused declaration will raise warning from static analyzer, and we won't wanna show a lot of warnings in PR or when someone reviewing, it might be better if you comment it out or remove it for now, since it is just a declaration, I believe it wont be hard to redeclare it next time.